Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -685,12 +772,24 @@ + (NSData *)exportLocalAuthenticatorCredentialWithGroupAndID:(NSString *)group c
     ]);
     updateQueryForGroupIfNecessary(query.get(), group);
 
+    if (useAlternateKeychainAttribute)
+        [query setObject:credentialID forKey:(id)kSecAttrAlias];
+    else
+        [query setObject:credentialID forKey:(id)kSecAttrApplicationLabel];
```

Comment:
Ditto my earlier comments about all of this duplicated code.

Reply:
Done